collectors/utils.py

```diff
@@ -116,6 +116,10 @@ def convert_cvss_score_to_impact(score: Union[Decimal, float]) -> Impact:
     This function converts CVSS score to Flaw Impact.
     Flaw Impact matches CVSS severity, which is defined by CVSS score ranges.
     """
+    # Ensure the score is always of Decimal type rounded to 1 decimal place
+    if not isinstance(score, Decimal):
+        score = round(Decimal(score), 1)
```

Comment:
Is it possible that we get `Decimal` instance but not rounded to 1 decimal place? Such case would pass unnoticed here. We can simply do:
```python
score = round(Decimal(score), 1)
```
as creating a new Decimal from Decimal and rounding it to the 1 place won't change it if it is already in the right format.

Reply:
`cvss` lib should always return numbers like `1.2`, but your suggestion is probably better. Thanks!